### DISCUSSION TITLE
[Feature]: Subtitle and audio delay (sync) support

### DISCUSSION THREAD

**Discussion by himanshu8443** [TheWidlarzGroup/react-native-video#4982] [Ideas]
### What problem are you trying to solve?

To sync unsynchronised TextTracks and audio tracks. Sometimes externally added TextTracks are out of sync, so it would be very helpful to be able to sync audio and caption tracks, especially TextTrack.

### Proposed solution

Add subtitle and audio delay options to sync audio and caption tracks.

[a comment by an automated account is not shown]

**Comment by freeboub** [comment]
It looks like a workaround for bad input file.
I had a quick check on exoplayer doc, I don't see such possibility... not sure this is feasible

**Comment by himanshu8443** [comment]
ok,  that's unfortunate.

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
